build-scripts/build-custom-app.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+# Define the content for the main.tsx file
+read -r -d '' MAIN_CONTENT <<EOM
+
+import React, { FC, ReactNode } from "react";
+import Header from "@components/headers/Main";
+import Footer from "@components/footer/Footer";
+
+type TProps = {
+  children?: ReactNode;
+};
+
+const Main: FC<TProps> = ({ children }) => (
+  <>
+    <Header />
+    <main id="main" className="flex flex-col flex-1">
+      {children}
+    </main>
+    <Footer />
+  </>
+);
+export default Main;
+
+EOM
+
+# Define the content for Analytics.tsx
+read -r -d '' ANALYTICS_CONTENT <<EOM
+
+import React from 'react';
+
+const Analytics = ({enableAnalytics}: {enableAnalytics: boolean}) => {
+    return null 
+};
+
+export default Analytics;
+
+EOM
+# Define the content for MethodologyLink.tsx
+read -r -d '' METHODOLOGY_LINK_CONTENT <<EOM
```

Comment:
I think we are going to want to copy the MethodologyLink.tsx page from `themes/cclw/components/MethodologyLink.tsx` and the corresponding methodology page from `themes/cclw/pages/methodology.tsx` instead of linking outwards to the CPR github page for custom apps - perhaps some template text in the methodology.tsx file that says we need to fill it in or whatever?